Fakebook.Posts/Fakebook.Posts.DataAccess/Models/Comment.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Fakebook.Posts.DataAccess.Models
+{
+    public class Comment
+    {
+        public int Id { get; set; }
+        public string UserEmail { get; set; }
+        public int PostId { get; set; }
```

Comment:
Without the data annotations the `public int PostId { get; set; }` property is completely pointless.

Reply:
that depends on what logic you want in the repositories. sometimes you don't need the whole entity, just the ID. it doesn't hurt one way or the other